security/gnupg-pkcs11-scd/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+
+name                    gnupg-pkcs11-scd
+version                 0.10.0
+
+categories              security
+platforms               darwin
```

Comment:
this is the default value for `platforms` since MacPorts v2.7 - please remove this line